protocol-units/execution/util/src/config/chain.rs

```diff
@@ -27,6 +27,9 @@ pub struct Config {
 	#[serde(default = "default_maptos_private_key")]
 	pub maptos_private_key: Ed25519PrivateKey,
 
+	#[serde(default)]
+	pub read_only: bool,
```

Comment:
The "chain" config is perhaps not a good name for this configuration section,
since we also have the `maptos_rest_*` fields here (the private key is here too, but it should be removed as per #387). Maybe we should move all these to a separate configuration section, or rename this one.

Reply:
By the current naming convention, this should be `maptos_read_only`, but see #480

Reply:
This is fine.